tests/python/hardware_test.py

```diff
@@ -1,174 +0,0 @@
-import argparse
```

Comment:
did you mean to delete this file?  it's not mentioned in your change summaries.

Reply:
yep, its intentional